src/common/components/common/TermsDialog.vue

```diff
@@ -1,25 +1,31 @@
 <template>
     <v-dialog v-model="show" width="650" :persistent="!areTermsAccepted">
-      <v-card class="container dialog">
-        <v-row class="mx-0 mt-6 d-flex justify-center">
-          <v-img :src="require('@/assets/common/terms-conditions.png')" height="60" contain />
+      <v-card class="container dialog pa-3">
+        <v-row class="d-flex justify-end mr-3 mt-3">
+          <v-btn variant="plain" :icon="mdiCloseCircleOutline"
+          size="regular" @click="close" >
+          </v-btn>
         </v-row>
-        <v-row class="mx-0 mt-6 mb-3 d-flex justify-center">
-          <h2>Terms and conditions</h2>
+        <v-row class="mx-0 mb-3 d-flex justify-center">
+          <div class="d-flex text-h3 ga-1 flex-wrap">
+            <span class='pa-2 bg-purple'>
+              Terms and conditions
+            </span>
+          </div>
         </v-row>
-        <v-row>
-          <v-container class="terms-txt mx-15 my-6 pa-10" @scroll="onScroll" ref="scrollableArea">
-            <v-row>
-              <p>{{ dialogText }}</p>
+        <v-row class="mx-15 my-6">
+          <v-container class="terms-txt" @scroll="onScroll" ref="scrollableArea">
+            <v-row class="pa-4 ma-0">
+              <p>Terms example</p>
```

Comment:
Replace by the actual value?

Reply:
@ronaldsg20 can you remove this text?

Reply:
The current value on the DB only stores if the terms are enabled or not, I'll update it in order to handle an additional field, but in plain text. then it will be available on the app